diff -Naur ndoutils-1.5.2/include/nagios-2x/nagios.h ndoutils-1.5.2-openbsd/include/nagios-2x/nagios.h
--- ndoutils-1.5.2/include/nagios-2x/nagios.h   Tue Apr 10 18:23:50 2007
+++ ndoutils-1.5.2-openbsd/include/nagios-2x/nagios.h   Sat Nov 23 17:13:09 2013
@@ -21,6 +21,7 @@
 #ifndef _NAGIOS_H
 #define _NAGIOS_H

+#include <pthread.h>
 #include "config.h"
 #include "common.h"
 #include "locations.h"
diff -Naur ndoutils-1.5.2/include/nagios-3x/nagios.h ndoutils-1.5.2-openbsd/include/nagios-3x/nagios.h
--- ndoutils-1.5.2/include/nagios-3x/nagios.h   Wed Oct 31 19:17:05 2007
+++ ndoutils-1.5.2-openbsd/include/nagios-3x/nagios.h   Sat Nov 23 17:13:17 2013
@@ -25,6 +25,7 @@
 # define __attribute__(x) /* nothing */
 #endif

+#include <pthread.h>
 #include "config.h"
 #include "common.h"
 #include "locations.h"
