diff -Naur ndoutils-1.4b9/include/nagios-2x/nagios.h ndoutils-1.4b9-openbsd/include/nagios-2x/nagios.h
--- ndoutils-1.4b9/include/nagios-2x/nagios.h   Tue Apr 10 18:21:35 2007
+++ ndoutils-1.4b9-openbsd/include/nagios-2x/nagios.h   Tue Oct 27 23:59:47 2009
@@ -21,6 +21,7 @@
 #ifndef _NAGIOS_H
 #define _NAGIOS_H

+#include <pthread.h>
 #include "config.h"
 #include "common.h"
 #include "locations.h"
diff -Naur ndoutils-1.4b9/include/nagios-3x/nagios.h ndoutils-1.4b9-openbsd/include/nagios-3x/nagios.h
--- ndoutils-1.4b9/include/nagios-3x/nagios.h   Wed Oct 31 19:17:05 2007
+++ ndoutils-1.4b9-openbsd/include/nagios-3x/nagios.h   Wed Oct 28 00:00:02 2009
@@ -25,6 +25,7 @@
 # define __attribute__(x) /* nothing */
 #endif

+#include <pthread.h>
 #include "config.h"
 #include "common.h"
 #include "locations.h"
